website/src/components/events/event-list-item.js

```diff
@@ -85,25 +85,37 @@ export default function EventListItem(props) {
     eventType = "Community Event"
   }
 
+  // Gatsby seems to hijack all <a> tags and, if they do not begin with http://
+  // or https://, appends the path to the domain instead of setting the URL as
+  // expected.  
+  let link = info.link
+  if (!link) link = info.where
+
+  // show link if there is a url available 
+  // TODO: is (.) dot sufficient to match any url? 
+  if (link.includes(".")) {
+    if (!link.includes("http")) link = "http://" + link
+    link = <Link to={link}>{info.where}</Link>
+  }
+
   return (
     <Card className={classes.card} elevation={0}>
       <CardContent className={classes.content}>
         <Typography variant="overline" style={{ color: isTNDWorkshop ? "#CC522C" : "#FAFFF7" }}>{eventType}</Typography>
         <Typography
           variant="h3"
           component="h3"
-          className={classes.title}
-        >
+          className={classes.title}>
           <Link to={props.path}>{info.title}</Link>
         </Typography>
-          <div>
-            <Typography variant="h6" className={classes.eventInfo}>
-              When: {formatDate(info.date)}
-            </Typography>
-            <Typography variant="h6" className={classes.eventInfo}>
-              Where: {info.where}
-            </Typography>
-          </div>
+        <div>
+          <Typography variant="h6" className={classes.eventInfo}>
+            When: {formatDate(info.date)}
+          </Typography>
+          <Typography variant="h6" className={classes.eventInfo}>
+            Where: {link}
```

Comment:
As mentioned above, let's keep `info.where` and `info.link` separate - they'll often contain different values, and so should be treated as different entities.